public/cli/lib/rendering/render.html

```diff
@@ -92,21 +92,17 @@
         }
 
         var subtitles = document.querySelector("#subtitles2");
```

Comment:
Avoid `var`. Prefer `const` for values that don't change and `let` for everything else. Here and below.